modules/backup-gcs/gcs/gcs.go

```diff
@@ -83,6 +84,8 @@ func (g *gcs) getObject(ctx context.Context, bucket *storage.BucketHandle,
 	if err != nil {
 		return nil, errors.Wrapf(err, "read object: %v", objectName)
 	}
+
+	monitoring.GetMetrics().BackupRestoreDataTransferred.WithLabelValues("backup-gcs", "class").Add(float64(len(content)))
```

Comment:
WithLabelValues might panic and we don't want to panic because of a metric.
You could use GetMetricWithLabelValues instead

Also might not be a good place to put metrics here since getObject and other methods here are class agnostic.